VK/VK/FriendPhotoScreen/Cell/PhotoCollectionViewCell.swift

```diff
@@ -16,23 +22,39 @@ final class PhotoCollectionViewCell: UICollectionViewCell {
 
     private var tapHandler: TapHandler?
     private var selectedFriend: NetworkUnit?
+    private var imagesURLStrings: [String] = []
+    private var images: [UIImage] = []
 
     // MARK: Public methods
 
     func configureCell(unit: NetworkUnit, handler: TapHandler?) {
-        tapHandler = handler
         selectedFriend = unit
+        getPhotos()
+        tapHandler = handler
         configureTapGestoreRecognizer()
-        configreAvatarImageView(imageNames: unit.unitImageNames ?? [])
     }
 
     // MARK: Private methods
 
-    private func configreAvatarImageView(imageNames: [String]) {
-        firstPhotoImageView.image = UIImage(named: imageNames[safe: 0] ?? "")
-        secondPhotoImageView.image = UIImage(named: imageNames[safe: 1] ?? "")
-        thirdPhotoImageView.image = UIImage(named: imageNames[safe: 2] ?? "")
-        foursPhotoImageView.image = UIImage(named: imageNames[safe: 3] ?? "")
+    private func configreAvatarImageView() {
+        for imagesURLString in imagesURLStrings {
+            NetworkServiceble.shared.fetchPhoto(by: imagesURLString) { [weak self] data in
+                guard let self = self else { return }
+                self.images.append(UIImage(data: data) ?? UIImage())
```

Comment:
```suggestion
                self.images.append(UIImage(data: data) )
```
Вместо непришедшей картинки лучше показывать заглушку, а не пустой UIImage

Reply:
Заменил по всему проекту в сл дз